### DISCUSSION TITLE
{.grid-list} wrong format

### DISCUSSION THREAD

**Discussion by TManOxO** [requarks/wiki#6172] [Error / Bug Report]
The version currently using: 2.5.297

![1](https://user-images.githubusercontent.com/49078539/219765550-bec6a2ab-1e98-4537-9677-a6cb81abd0e6.png)
![2](https://user-images.githubusercontent.com/49078539/219765564-40139f0c-7167-4883-8404-c674cbc7bc6d.png)

Official website description
![3](https://user-images.githubusercontent.com/49078539/219765996-f3c5f9e8-6fd5-4e77-83b9-c60e88d5cb8d.png)

**Comment by NGPixel** [comment]
You can't use 2 lists with different styles back to back. You need an element to separate them. The example in the docs is meant just to show the syntax to use, not a real world example.
